packages/picasso.js/src/core/chart-components/labels/strategies/__tests__/bars.spec.js

```diff
@@ -253,6 +255,22 @@ describe('labeling - bars', () => {
       expect(p.bounds).to.equal(rects.biggest);
     });
 
+    it('should find largest rect as fallback, horizontal, and when all rects have the same height', () => {
+      rects.inside.height = rects.outside.height = rects.biggest.height = rects.meh.height = 100;
```

Comment:
Don't use this kind of syntax. Declare a const with desired value and assign it to each property on a separate row.

Reply:
Fixed!